test/integration/test_push_docker_image.py

```diff
@@ -91,19 +107,14 @@ def docker_registry(request):
         container.remove()
 
 
-def test_push_tag(dss_docker_image, docker_registry):
-    repo = dss_docker_image.repository
-    tag = "999.9.9"
-    with tagged_image(dss_docker_image, docker_registry, tag) as tagged:
-        docker_registry.push(tagged.repository, tag)
-    assert repo in docker_registry.repositories
-    assert tag in docker_registry.images(repo)["tags"]
-
-
-def test_push_via_image(dss_docker_image, docker_registry):
-    repo = dss_docker_image.repository
-    tag = dss_docker_image.version
-    with tagged_image(dss_docker_image, docker_registry) as tagged:
-        tagged._push()
+def test_push_tag(sample_docker_image, docker_registry):
+    old = DockerImageSpec("registry", "2")
```

Comment:
I think, you don't need this, because you are getting samle_docker_image as input